tools/test/test_get_container_images.py

```diff
@@ -0,0 +1,187 @@
+import pytest
+from pytest_mock import MockerFixture
+
+from reconcile.gql_definitions.common.namespaces_minimal import ClusterV1, NamespaceV1
+from reconcile.gql_definitions.fragments.vault_secret import VaultSecret
+from reconcile.test.fixtures import Fixtures
+from reconcile.utils.oc import OCNative
+from reconcile.utils.oc_map import OCMap
+from tools.cli_commands.container_images_report import (
+    _fetch_pods_images_from_namespaces,
```

Comment:
can remove `_` prefix in function name as this is the main tested public one.

Reply:
sure, I was hesitant since I only created that function for testing, but it makes sense.